So it looks like the build fell over but for some reason it didn't cause the job to exit. The build error is buried in the console log:

```
C:\Jenkins\workspace\pull_requests-conda-windows\qt\scientific_interfaces\ISISReflectometry\GUI\Preview\PreviewPresenter.cpp(171,36): error C2039: 'Strings': is not a member of 'Mantid::Kernel' [C:\Jenkins\workspace\pull_requests-conda-windows\build\qt\scientific_interfaces\ISISReflectometry\MantidScientificInterfacesISISReflectometryQt5.vcxproj]
C:\Jenkins\workspace\pull_requests-conda-windows\Framework\Kernel\inc\MantidKernel/make_cow.h(18): message : see declaration of 'Mantid::Kernel' [C:\Jenkins\workspace\pull_requests-conda-windows\build\qt\scientific_interfaces\ISISReflectometry\MantidScientificInterfacesISISReflectometryQt5.vcxproj]
C:\Jenkins\workspace\pull_requests-conda-windows\qt\scientific_interfaces\ISISReflectometry\GUI\Preview\PreviewPresenter.cpp(171,36): error C3083: 'Strings': the symbol to the left of a '::' must be a type [C:\Jenkins\workspace\pull_requests-conda-windows\build\qt\scientific_interfaces\ISISReflectometry\MantidScientificInterfacesISISReflectometryQt5.vcxproj]
C:\Jenkins\workspace\pull_requests-conda-windows\qt\scientific_interfaces\ISISReflectometry\GUI\Preview\PreviewPresenter.cpp(171,45): error C2039: 'simpleJoin': is not a member of 'Mantid::Kernel' [C:\Jenkins\workspace\pull_requests-conda-windows\build\qt\scientific_interfaces\ISISReflectometry\MantidScientificInterfacesISISReflectometryQt5.vcxproj]
C:\Jenkins\workspace\pull_requests-conda-windows\Framework\Kernel\inc\MantidKernel/make_cow.h(18): message : see declaration of 'Mantid::Kernel' [C:\Jenkins\workspace\pull_requests-conda-windows\build\qt\scientific_interfaces\ISISReflectometry\MantidScientificInterfacesISISReflectometryQt5.vcxproj]
C:\Jenkins\workspace\pull_requests-conda-windows\qt\scientific_interfaces\ISISReflectometry\GUI\Preview\PreviewPresenter.cpp(171,55): error C3861: 'simpleJoin': identifier not found [C:\Jenkins\workspace\pull_requests-conda-windows\build\qt\scientific_interfaces\ISISReflectometry\MantidScientificInterfacesISISReflectometryQt5.vcxproj]
```

I'll look at why the build didn't just stop the job.

--- a/qt/scientific_interfaces/ISISReflectometry/GUI/Experiment/ExperimentPresenter.cpp
+++ b/qt/scientific_interfaces/ISISReflectometry/GUI/Experiment/ExperimentPresenter.cpp
@@ -108,7 +108,7 @@ void ExperimentPresenter::notifyPreviewApplyRequested(PreviewRow const &previewR
   if (auto const foundRow = m_model.findLookupRow(previewRow, m_thetaTolerance)) {
     auto lookupRowCopy = *foundRow;
 
-    lookupRowCopy.setRoiDetectorIDs(std::move(previewRow.getSelectedBanks()));
+    lookupRowCopy.setRoiDetectorIDs(previewRow.getSelectedBanks());
 
     updateLookupRowProcessingInstructions(previewRow, lookupRowCopy, ROIType::Signal);
     updateLookupRowProcessingInstructions(previewRow, lookupRowCopy, ROIType::Background);

--- a/qt/scientific_interfaces/ISISReflectometry/GUI/Preview/PreviewPresenter.cpp
+++ b/qt/scientific_interfaces/ISISReflectometry/GUI/Preview/PreviewPresenter.cpp
@@ -9,6 +9,7 @@
 #include "Common/Detector.h"
 #include "GUI/Batch/IBatchPresenter.h"
 #include "MantidAPI/MatrixWorkspace.h"
+#include "MantidKernel/Strings.h"
 #include "MantidQtWidgets/Plotting/AxisID.h"
 #include "MantidQtWidgets/RegionSelector/IRegionSelector.h"
 #include "MantidQtWidgets/RegionSelector/RegionSelector.h"

